file2.txt

```diff
@@ -0,0 +1,2 @@
+La
+took benefit of LaLa feature to make LaLaLa
```

Comment:
This looks really cool. Can you please put some comment around your code

Reply:
Been there done that